README.markdown

```diff
@@ -24,3 +24,51 @@ Quero ajudar!
 
 [![Bitdeli Badge](https://d2weczhvl823v0.cloudfront.net/rennerocha/dojopuzzles/trend.png)](https://bitdeli.com/free "Bitdeli Badge")
 
+Rodando o projeto!
+----------------------
+
+### Requeriments
+
+   - Python 3.7
+   - Django 3
+
+### Instalando a VirtualEnv
```

Comment:
**suggestion**: Instead of suggesting a specific tool to create a separated virtualenv, just add a suggestion that it would be a good idea to create a separated virtual environment before installing the project. I use `pyenv` to manage them for example.